xml/System/InvalidOperationException.xml

```diff
@@ -202,8 +202,8 @@
   
  The following example throws an <xref:System.InvalidOperationException> exception  when it attempts to iterate an array that includes a `Nullable(Of Integer)` value.  
   
- [!code-csharp[System.InvalidOperationException#4](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.invalidoperationexception/cs/Nullable1.cs#4)]
- [!code-vb[System.InvalidOperationException#4](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.invalidoperationexception/vb/Nullable1.vb#4)]  
+ :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.invalidoperationexception/cs/Nullable1.cs" interactive="try-dotnet" id="Snippet4":::
```

Comment:
so this was number 14 on my count, so we also need to enable the interactive on the previous one, which means the next ones are off by 1 too

```suggestion
 :::code language="csharp" source="~/samples/snippets/csharp/VS_Snippets_CLR_System/system.invalidoperationexception/cs/Nullable1.cs" id="Snippet4":::
```